src/transformers/dependency_versions_table.py

```diff
@@ -66,4 +66,5 @@
     "unidic": "unidic>=1.0.2",
     "unidic_lite": "unidic_lite>=1.0.7",
     "uvicorn": "uvicorn",
+    "detectron2" : "detectron2",
```

Comment:
This has to be `"detectron2 @ git+https://github.com/facebookresearch/detectron2.git": "git+https://github.com/facebookresearch/detectron2.git@v0.4#egg=detectron2",` not `"detectron2" : "detectron2",`

Reply:
done still erroring out . any thought ?

Reply:
This isnt changed? my comment said for this to be  `"detectron2 @ git+https://github.com/facebookresearch/detectron2.git": "git+https://github.com/facebookresearch/detectron2.git@v0.4#egg=detectron2",`

Reply:
Oh, it's still failing? Weird, mine builds locally totally fine